api/base/v1alpha1/datasource_types.go

```diff
@@ -54,6 +54,37 @@ type OSS struct {
 	Object string `json:"object,omitempty"`
 }
 
+// PostgreSQL defines info for PostgreSQL
+//
+// ref: https://github.com/jackc/pgx/blame/v5.5.1/pgconn/config.go#L409
+// they are common standard PostgreSQL environment variables
+// For convenience, we use the same name.
+//
+// The PGUSER/PGPASSWORD/PGPASSFILE/PGSSLPASSWORD parameters have been intentionally excluded
+// because they contain sensitive information and are stored in the secret pointed to by `endpoint.authSecret`.
+type PostgreSQL struct {
+	Host               string `json:"PGHOST,omitempty"`
```

Comment:
@Abirdcfly Can we reuse the content in `Endpoint`?

Reply:
We reuse endpoint, and here is just a list of all the other parameters that we can use.

Reply:
So how they (endpoit.url and Posgresql.Host ) work together ?

Reply:
set `endpoint.url` to  `pgx.Connect(ctx, connString)`, and set PostgreSQL.Host to env `PGHOST` (other PostgreSQL.xxx to env PGxxx), pgx (or other pg go library) will merge settings like `mergeSettings(defaultSettings, envSettings, connStringSettings)` (https://github.com/jackc/pgx/blob/6f2ce92356f35263258ef7baa4cde86a3e1c6255/pgconn/config.go#L250)